index.html

```diff
@@ -49,98 +61,13 @@ <h2>I am a Software Developer</h2>
                 <li><a href="#"><i class="fa fa-medium"></i></a></li>
             </ul>
         </section>
+
         <!-- Branch 2: Work and About me -->
         <section class="main_middle_section" id="portfolio">
```

Comment:
The JS part is very good. But kindly follow the Figma template for margins, paddings, font-sizes, colors etc. 
<img width="1224" alt="Screenshot 2022-12-22 at 12 36 44 AM" src="https://user-images.githubusercontent.com/25887253/209007840-c6df0f77-2530-47d9-a9c3-1f22d3eeb790.png">

Reply:
- [ ] Hi Good job so far but the previous reviewer pointed out some parts of your design. Unfortunately, I can see only half of your pop windows I'm not able to confirm you fix your design as required by the previous reviewer because I can not scroll up to your pop is fixed I can see only half. Please make sure that we are able to see all your popup with the design fixed as required by the previous reviewer and note that it is very important to stick the Figma design as much as possible :+1: 

#### Please check the shared video to have more details
https://www.loom.com/share/1a4cfc0950294c6cb53978ec213c54ce